aisimulate/src/aisimulate/spica/__init__.py

```
@@ -20,95 +34,92 @@
+
+def __getattr__(name: str) -> Any:
+    target = _LAZY_EXPORTS.get(name)
+    if target is None:
+        raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
+    module_name, attribute = target
+    value = getattr(importlib.import_module(module_name, __name__), attribute)
+    globals()[name] = value
+    return value
+
 
 __all__ = [
```

Ruff (0.16.0)

[error] Invalid object in `__all__`, must contain only strings

(PLE0604)